sc2tap/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM cadc-tomcat:macos
```

Comment:
cadc:tomcat:1 ??

Reply:
Oops, my custom cadc-tomcat build that allows writing of files inside the container. Fixed.

Reply:
I did some work to move that config file into ~/config instead of inside the war file; with that, the deployer can set the baseDir to something writable like /var/tmp/sc2tap/ (eg) and it will work out of the box... allows dev config to differ from prod without changing files that are in version control. I'll merge that in once this is done.